platform/security/sts/security-sts-casclaimshandler/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.2.0">
+
+  <OCD description="CAS Claims Handler Configuration"
+    name="Security STS CAS Claims Handler"
+    id="org.codice.ddf.security.claims.cas.CasClaimsHandler">
+
+    <AD name="Attribute Map:" id="attributeMap" required="true" type="String" cardinality="100"
+      default=""
+      description="List of key-value pairs, each mapping a supported claim to the CAS attribute
+                   it should be assigned. Entries must be of the form claim=attribute, and equals
+                   signs in the claim must be escaped.">
+    </AD>
+
+    <AD name="Supported Claims:" id="supportedClaims" required="true" type="String" cardinality="100"
```

Comment:
❓ Does this need to be configurable? Can we get the keys from the attributemapping on the backend?

Reply:
Ah, yeah that should work. Thanks for the suggestion.